src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AndroidBroker.cs

```diff
@@ -37,63 +37,55 @@
 using Microsoft.Identity.Core.Cache;
 using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.OAuth2;
 using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Helpers;
+using Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Broker;
 
 namespace Microsoft.IdentityModel.Clients.ActiveDirectory.Internal.Platform
 {
     [Android.Runtime.Preserve(AllMembers = true)]
-    internal class BrokerHelper
+    internal class AndroidBroker : IBroker
```

Comment:
Much better than having all of the BrokerHelper classes with different implementations. easier to find stuff.